src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -117,6 +131,7 @@ public ConfigurationManager(string metadataAddress, IConfigurationRetriever<T> c
             MetadataAddress = metadataAddress;
             _docRetriever = docRetriever;
             _configRetriever = configRetriever;
+            EnsureBackgroundTaskIsRunning();
```

Comment:
If the AppContextSwitch directs the runtime to use blocking mode, should we start the task here?